get-started/searchgov-for-cloudgov-pages.md

```diff
@@ -0,0 +1,57 @@
+---
+last_modified_at: October 28, 2022
+crumbname: Search.gov for Cloud.gov Pages
+layout: page
+title: Setting up Search.gov for Cloud.gov Pages sites
+redirect_from:
+  - /manual/searchgov-for-federalist.html
+  - /get-started/searchgov-for-federalist.html
+sidenav: get_started
+date: April 22, 2019
+tags: getting-started go-live site-launch
+description: Detailed documentation on integrating Search.gov with Cloud.gov Pages websites.
+---
+For over 20 years, GSA's Search.gov has transformed the public’s search experience on federal government websites. Agencies use our free, shared service to power over 2,000 search boxes on over 30% of federal domains. This page will walk you through the steps required to integrate Search.gov with your Cloud.gov Pages website. Examples are for websites using the Jekyll site template.
+
+You can get everything set up with us before launching your website, except for actually indexing your content. We will index your pages after the website is live.
+
+If you wish to test before launch, you'll need to publish the website to a publicly accessible domain such as `demo.example.gov`. You'll work with your agency's DNS team to set this up, and then follow the instructions below.
+
+## On the Cloud.gov Pages side of things, part 1
+
+1. Open your `_config.yml` file in your Cloud.gov Pages repo. In the `plugins` section, confirm that the `jekyll-sitemap` gem is in the list. For more information, [read the docs here](https://github.com/jekyll/jekyll-sitemap) and the [instructions for GitHub Pages here](https://help.github.com/en/articles/sitemaps-for-github-pages).
+2. Discuss and decide which search results page to use. We recommend you use our hosted search results page. You can also pull search results into your website for display. Please note that at this time, pulling search results into your website will result in an incomplete set of search analytics.
+
+## On the Search.gov side of things:
+
+1. Sign up for a [user account](https://search.usa.gov/signup).
+2. Read our \[Search Site Launch Guide]({{ site.baseurl }}/manual/site-launch-guide.html) if you’d like some direction.
```

Comment:
NetlifyCMS was trying to kill me. It kept jumping the cursor to the end of the paragraph after each character I typed. I switched to richtext mode hoping it would be different, but it was not. Any idea why it might have been doing that?

Reply:
Not sure, I would have to investigate. Can you make these changes via code for now? I can make an item in my backlog to retest netlifyCMS.

Reply:
Yeah, I bailed out to local after doing just one file.

Reply:
seems like a known bug - https://github.com/netlify/netlify-cms/issues/5092 will try the recommended CSS fix